jscomp/ml/matching.ml

```diff
@@ -26,6 +26,19 @@ open Printf
 
 let dbg = false
 
+let find_name (attr : Parsetree.attribute) =
```

Comment:
This is similar to `find_name` in `jscomp/core/record_attributes_check.ml`. I prefer if this can be extracted to a common place where both places can use the same function, not sure where though.

Reply:
Looks identical. Can't you use the one from `record_attributes_check.ml`?

Reply:
> Looks identical. Can't you use the one from `record_attributes_check.ml`?

I tried, but `record_attributes_check.ml` is in the `core` package, and trying to import it from `ml` causes a circular import.

Reply:
> I tried, but `record_attributes_check.ml` is in the `core` package, and trying to import it from `ml` causes a circular import.

Sorry I'm an idiot, I can reverse the dependency 🤦